sshcode.go

```diff
@@ -93,11 +101,12 @@ func sshCode(host, dir string, o options) error {
 		return xerrors.Errorf("failed to find available remote port: %w", err)
 	}
 
-	flog.Info("Tunneling local port %v to remote port %v", o.localPort, o.remotePort)
+	flog.Info("Tunneling local host %v:%v to remote port %v", bindHost, bindPort, o.remotePort)
```

Comment:
I'd probably reword to say `Tunneling remote port %v to %v, o.remotePort, bindAddr`